forge-game/src/main/java/forge/game/combat/GlobalAttackRestrictions.java

```diff
@@ -19,9 +20,11 @@
 public class GlobalAttackRestrictions {
 
     private final int max;
+    private final int min;
```

Comment:
This looks wrong, it's a requirement but you're trying to model it as a restriction

_Trove of Temptation_ does that better, but we need to research if it's a single requirement on a player or on every creature

it matters when comparing which attack is legal

Reply:
Ah, I see the distinction.

Reply:
It definitely reads like it's a requirement on the player... there is a note that says the Trove also is:
![image](https://user-images.githubusercontent.com/103371817/193683121-b079d91d-4490-4770-a14f-68709df8717a.png)